### PR TITLE
Kubernetes upgrades to v1.25

### PR DESCRIPTION
**Pull request by ronardcaktus**
Closes:
- https://app.clickup.com/t/86774uj0w

### Summary

Performs Kubernetes upgrades in the traffic stops cluster. Updates the following:

Kubernetes and Nodes -> `1.25`
Kubectl -> `1.25.10`
Helm -> `3.12.0`
Descheduler -> `0.25`
Ansible requirements
Other minor package dependencies

### CODE REVIEW

**Review by Afani97 — APPROVED** [review]
Looks good to me 👍

File: deploy/group_vars/all.yml [hunk 1]
```diff
@@ -96,7 +96,7 @@ k8s_iam_users: [copelco]
 # https://github.com/kubernetes/ingress-nginx/releases
 k8s_ingress_nginx_chart_version: "4.4.2"
```
Comment (tobiasmcnulty) [inline deploy/group_vars/all.yml:97]: Is there a reason to leave this at 4.4.2?

**Review by tobiasmcnulty — APPROVED** [review]
One nitpick, otherwise lgtm! 🚀

File: deploy/group_vars/all.yml (hunk 1 above)
Comment (ronardcaktus) [reply deploy/group_vars/all.yml:97]: I remember Colin and I talking about it, but don't remember the specific reason.

**Pull request merged**